test/end-to-end/cases/xspec-report.xspec

```diff
@@ -4,11 +4,25 @@
 	xquery-version="3.1" xslt-version="3.0">
 
 	<x:scenario label="Function (xspec/xspec#355)">
-		<!-- function-lookup() requires Saxon-PE or EE. So use a map. -->
-		<x:call function="exactly-one">
-			<x:param as="map(*)" select="map { 'foo': 1 }" />
-		</x:call>
-		<x:expect label="[Result] should be reported" />
+		<!-- function(*) (excluding array(*) and map(*)) requires Saxon-PE or EE -->
+
+		<x:scenario label="Array">
+			<x:call function="exactly-one">
+				<x:param as="array(*)" select="['foo', 1, [2, 'bar']]" />
+			</x:call>
+			<x:expect label="[Result] should be reported" />
```

Comment:
Would you consider a more descriptive label, such as "Serialized array (XSLT, assuming Saxon-PE/EE) or empty 'pseudo-other' element (XQuery) should be reported upon failure"?

I know this label predates this pull request, so feel free to disregard this comment or recommend a new enhancement for it.

Reply:
Good point. I updated this PR based on your suggestion: 85d0b245dd04bf6d1b89227c2d3ebf49361584ca

Reply:
Thanks!